src/components/Popover/PopoverItems.tsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+
+import React from "react";
+import Link from "next/link";
+
+import cc from "classcat";
+
+import { Icon, IconType } from "@/components/Icon/Icon";
+
+export type PopoverItemsProps = {
+  className?: string;
+  text: string;
+  subText?: string;
+  href?: string;
+  icon: IconType;
+  onClick?: () => void;
+  isTopBorder?: boolean;
+};
+
+export const PopoverItems: React.FC<PopoverItemsProps> = (props) => {
+  const itemClass = cc([
+    "whitespace-nowrap p-2 text-medium text-gray",
+    {
+      "border-t border-lightGray": props.isTopBorder,
+    },
+  ]);
+  return (
+    <li key={props.text} className={itemClass}>
```

Comment:
すんません。ここはkey要らないですね。
```suggestion
    <li className={itemClass}>
```